### DISCUSSION TITLE
Pull request accross different repository

### DISCUSSION THREAD

**Discussion by parteekcoder** [PyGithub/PyGithub#2539] [Q&A]
Can Someone Help me to figure this out

I have 2 repositories

repo1 = Organization/project
repo2 = user/project (it is forked of repo1)

Can I create a Pull Request from repo2 to repo1 using a Fine-Grained token with the required access of repo2 using PyGithub, If yes can you please help me to do that

Here is what I am doing:

g = Github("token") (token of repo2)
repo1 = g.get_repo("organization/project")
repo2 = g.get_repo("user/project")
repo1.create_pull(title="PR_TITLE", body="PR_BODY", base="main",head="user/project:main")

I am getting this error
```
github.GithubException.GithubException: 403 {"message": "Resource not accessible by personal access token", "documentation_url": "https://docs.github.com/rest/reference/pulls#create-a-pull-request"}
```

**Comment by EnricoMi** [comment]
see #2538
